### ISSUE TITLE
how to get logits and cell information for wikitable_qa?

### ISSUE THREAD

**Issue by sujit420** [microsoft/Table-Pretraining#14]
how to get logits and cell information for wikitable_qa? @SivilTaram @microsoftopensource 
Any help is really appreciated.

**Comment by SivilTaram** [comment]
@sujit420 Hi @sujit420, sorry for the late reply. If I understand correctly, you want the model to explicitly give which cell the answer comes from, right? If it is that, the current TAPEX cannot give such evidence, since it follows an end-to-end generation manner. Any follow-up question is warmly welcome!

**Comment by sujit420** [comment]
> @sujit420 Hi @sujit420, sorry for the late reply. If I understand correctly, you want the model to explicitly give which cell the answer comes from, right? If it is that, the current TAPEX cannot give such evidence, since it follows an end-to-end generation manner. Any follow-up question is warmly welcome!

any help on getting logits information?

**Comment by SivilTaram** [comment]
@sujit420 Maybe you can try to obtain logits from Huggingface's pre-trained models as [here](https://huggingface.co/microsoft/tapex-large-finetuned-wtq).

**Comment by SivilTaram** [comment]
@sujit420 for this repo, I would suggest modifying the code at https://github.com/microsoft/Table-Pretraining/blob/0cb07585703718db6afd568091ef556727bcb0b0/tapex/model_interface.py#L39 to return both the results and the answer. And then use the script at https://github.com/microsoft/Table-Pretraining/tree/main/examples#-interact to obtain logits.
